src/services/config.js

```diff
@@ -27,3 +27,8 @@ if (admin_pass) {
     pass: admin_pass
   }
 }
+
+config.auth_enabled = {
```

Comment:
The values are. They are configured as strings because all env variables are strings.